shoplist/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
```

Comment:
For release variant, we can have minifyEnabled as true and should have a buildType for debug where we can have it as false. 
In release tag we can also set debuggable to false as well
We can add "shrinkResources" tag as well, which can help with the built bundle size